services/wallet/router/fees.go

```diff
@@ -330,7 +329,12 @@ func (f *FeeManager) getFeeHistorySorted(chainID uint64) ([]*big.Int, error) {
 	return fees, nil
 }
 
-func (f *FeeManager) GetL1Fee(ctx context.Context, chainID uint64, tx *ethTypes.Transaction) (uint64, error) {
+// Returns L1 fee for placing a transaction to L1 chain, appicable only for txs made from L2.
+func (f *FeeManager) GetL1Fee(ctx context.Context, chainID uint64, input []byte) (uint64, error) {
+	if chainID == common.EthereumMainnet || chainID == common.EthereumSepolia && chainID != common.EthereumGoerli {
```

Comment:
that's only valid if chain id is optimism like chain no?

Reply:
For optimism for sure, for arbitrum it gives 0, so no problem.